python/paddle/fluid/tests/unittests/test_fc_op.py

```diff
@@ -131,6 +133,34 @@ def config(self):
         self.matrix = MatrixGenerate(1, 4, 3, 128, 128, 2)
 
 
+class TestFcAPI(unittest.TestCase):
+    # test for parameter num_flatten_dims=-1
+    def test_api(self):
+        startup_program = Program()
+        main_program = Program()
+        startup_program.random_seed = SEED
+        main_program.random_seed = SEED
+
+        with program_guard(main_program, startup_program):
+            input = np.random.random([2, 2, 25]).astype("float32")
+            x = fluid.layers.data(
+                name="x",
+                shape=[2, 2, 25],
+                append_batch_size=False,
+                dtype="float32")
+
+            out_1 = fluid.layers.fc(input=x, size=1, num_flatten_dims=-1)
+            out_2 = fluid.layers.fc(input=x, size=1, num_flatten_dims=2)
+
+        exe = fluid.Executor(place=fluid.CPUPlace())
```

Comment:
Could you write something like:

exe = fluid.Executor(fluid.CPUPlace(
            ) if not core.is_compiled_with_cuda() else fluid.CUDAPlace(0))

It tests GPU case with minor modification.

Reply:
Thanks. Done